client/state/receipts/Makefile

```diff
@@ -0,0 +1,10 @@
+REPORTER ?= spec
```

Comment:
Just an idea. We keep adding tests in subdirectories, and our tests keep getting slower to run. Could we add this to /client and start to build something that can run all the tests together? Probably should be for another PR, but it was just a thought.